scst-store/multicluster-setup.hbs.md

```diff
@@ -156,17 +145,25 @@ certificate and edit token.
     kubectl create ns amr-observer-system
     ```
 
-1. Apply the CloudEvent Handler CA certificate `ceh_ca.yaml` secret YAML you generated earlier.
+1. Update values.yaml file and set CloudEvent Handler CA certificate data in `amr.observer.ca_cert_data` field.
 
     ```console
-    kubectl apply -f ceh_ca.yaml
+    amr:
+      observer:
+        auth:
+          kubernetes_service_accounts:
+            enable: true
+        cloudevent_handler:
+          endpoint: https://amr-cloudevent-handler.ingress-domain
```

Comment:
Shouldn't `ingress-domain` be in all caps? I feel like things that need to be changed by the user has been in all-caps and a section to describe what it is.

Reply:
Updated